models/bamf_nnunet_mr_liver/utils/BamfProcessorRunner.py

```diff
@@ -0,0 +1,66 @@
+"""
+-------------------------------------------------
+MHub - Run Module for ensembling nnUNet inference.
+-------------------------------------------------
+-------------------------------------------------
+Author: Rahul Soni
+Email:  rahul.soni@bamfhealth.com
+-------------------------------------------------
+"""
+
+from mhubio.core import Instance, InstanceData, DataType, FileType, CT, SEG
+from mhubio.core import Module, IO
+import os, numpy as np
+import SimpleITK as sitk
+from skimage import measure, filters
+import numpy as np
+import shutil
+
+
+
+class BamfProcessorRunner(Module):
+
+    @IO.Instance
+    @IO.Input('in_data', 'nifti:mod=ct|mr', the='input data to run nnunet on')
+    @IO.Output('out_data', 'bamf_processed.nrrd', 'nrrd:mod=seg:processor=bamf', data='in_data', the="keep the two largest connected components of the segmentation and remove all other ones")
+    def task(self, instance: Instance, in_data: InstanceData, out_data: InstanceData) -> None:
+       # Log bamf runner info
+        self.v("Running BamfProcessor on....")
```

Comment:
As this file is used in all four models (and potential future models I suppose), I suggest to use the `self.log` function instead of the legacy `self.v`. It works pretty much the same but gives some additional control over the log-level. Documentation can be found here: https://github.com/MHubAI/documentation/blob/main/documentation/mhubio/how_to_write_an_mhubio_module.md#log-messages

Reply:
Simply replacing `self.v` by `self.log` is already sufficient here!